source/ceylon/toml/formatToml.ceylon

```diff
@@ -87,6 +87,17 @@ shared String formatToml(Map<String, Object> tomlTable)
         emitBasicString(item);
     }
 
+    "Is the character in the Unicode 'private use' category?"
+    function privateUse(Character char)
+        =>     '\{#E000}' <= char <= '\{#F8FF}'
+            || '\{#0F0000}' <= char <= '\{#0FFFFD}'
+            || '\{#100000}' <= char <= '\{#10FFFD}';
+
+    "Is the character in the Unicode 'control' category?"
+    function control(Character char)
```

Comment:
I'm not a character set expert so I don't know how `Character.control` should best be documented. My complaint as a non-expert is that it took me a couple google searches and code lookups to determine that

> Determine if this character is an ISO control character

is effectively the same as

> Determine if this character is in the unicode category Other, Control (`Cc`)

Regardless, the utility function in this PR is not exported, so I think it's fine.